spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/audit/domain/AuditOperationType.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.audit.domain;
+
+import org.springframework.util.Assert;
+
+/**
+ * The application area an {@link AuditRecord} is associated with.
+ *
+ * @author Gunnar Hillert
+ *
+ */
+public enum AuditOperationType {
+
+	STREAM_DEFINITIONS(100L, "Stream Definitions"),
+	TASK_DEFINITIONS(  200L, "Task Definitions"),
+	APP_REGISTRATIONS( 300L, "App Registrations"),
+	SCHEDULES(         400L, "Schedules");
+
```

Comment:
`stream', `task`, 'app_registration` (or maybe just `registration`), and `schedule` would be more descriptive/appropriate - reads better

Reply:
Done with the latest commit